companion-packages/meteorrn-local/index.js

```diff
@@ -96,11 +96,12 @@ const Local = {
 
       LiveCol.find({}).observe({
         added: async (doc) => {
-          LocalCol._collection.upsert(doc);
+          LocalCol.insert(doc);
```

Comment:
With your _stale items, won't this leave duplicates of all the items in the collection temporarily? This doesn't seem like it makes anything better, just more unpredictable with a timeout

Reply:
Yeah, I totally discarded this approach actually. If it's ok I'd rather close this PR, this is not leading to any good. I'm really happy with how #128 is working. Much more predictable results that way.

Reply:
Yep makes sense, that approach seems to add much fewer edge cases